php/classes/controllers/class-settings-controller.php

```diff
@@ -785,7 +790,8 @@ public function settings_page() {
 			);
 			$html .= '<form method="post" action="' . esc_url_raw( $current_admin_url ) . '" enctype="multipart/form-data">' . "\n";
 			$html .= '<input type="hidden" name="action" value="post_import_form" />';
-			$html .= wp_nonce_field( 'ss_podcasting_import' );
+			$html .= wp_nonce_field( 'ss_podcasting_import', '_wpnonce', true, false );
```

Comment:
Fixed a bug - if the 4th parameter is empty, it prints the nonce field, so we had 2 nonce fields from one call - one when it fires, and one when it $html is being printed.